roles/custom/matrix-jitsi/templates/jicofo/env.j2

```diff
@@ -27,7 +27,7 @@ JIGASI_SIP_URI
 JVB_BREWERY_MUC={{ matrix_jitsi_jvb_brewery_muc }}
 MAX_BRIDGE_PARTICIPANTS
 OCTO_BRIDGE_SELECTION_STRATEGY
-SENTRY_DSN="${JICOFO_SENTRY_DSN:-0}"
+SENTRY_DSN={{1 if matrix_jitsi_jicofo_sentry_dns else 0}}
```

Comment:
Should this be a true/false value or the actual Sentry DSN address or something else?

Reply:
It's expecting it to be 0 or 1 which then gets converted to a boolean

Reply:
So basically it ends up setting this value: https://github.com/jitsi/docker-jitsi-meet/blob/c44c59e6cb5ccdadfe0e710b7c649545fb5904a8/jvb/rootfs/defaults/logging.properties#L1

I wonder if a better name would be "matrix_jitsi_jicofo_sentry_dns_enabled" with a default of "0"

Reply:
I think that following the upstream names is more consistent with how we handle things in other roles. Would you revert [4146c48](https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/2274/commits/4146c48f6a2e71d1b0d3f58c767aea1b2f4f789c)?

Reply:
ok cool